lib/pf/access_filter/radius.pm

```diff
@@ -79,6 +79,17 @@ sub handleAnswerInRule {
         if (defined($rule->{'merge_answer'}) && !(isenabled($rule->{'merge_answer'}))) {
             return ($radius_reply,$status);
         } else {
+            foreach my $key (keys %$radius_reply_ref) {
+                if (exists($radius_reply->{$key})) {
+                    if (defined(reftype($radius_reply->{$key})) && reftype($radius_reply->{$key}) eq 'ARRAY') {
+                        my @attribute;
+                        push(@attribute,@{$radius_reply_ref->{$key}});
```

Comment:
If it's not an array ref then it mean that the guy who wrote the switch module did something wrong.